src/app/features/location/geo.service.ts

```diff
@@ -87,9 +87,19 @@ export class GeoService {
    * @param coordinates of a place (`lat` and `lon`)
    */
   reverseLookup(coordinates: Coordinates): Observable<GeoResult> {
+    if (!coordinates) {
+      return undefined;
+    }
+
+    const fallback: GeoResult = {
+      display_name: $localize`[selected on map: ${coordinates.lat} - ${coordinates.lon}]`,
```

Comment:
```suggestion
      display_name: $localize`[selected coordinates: ${coordinates.lat} - ${coordinates.lon}]`,
```

generalizing this a bit to not confuse people who used GPS location